types/jquery.fancytree/index.d.ts

```diff
@@ -835,7 +838,7 @@ declare namespace Fancytree {
         /** Animation options, false:off (default: { effect: "blind", options: {direction: "vertical", scale: "box"}, duration: 200 }) */
         toggleEffect?: JQueryUI.EffectOptions;
         /** Tooltips */
-        tooltip?: boolean;
+        tooltip?: boolean | string | ((event: JQueryEventObject, data: EventData) => string|boolean);
```

Comment:
According to the docs you linked, `string` isn’t a valid type here. (Same goes for `iconTooltip`.)